src/model/LabyrinthMatrix.java

```diff
@@ -1,73 +0,0 @@
-package model;
-import java.io.File;
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Scanner;
-
-public class LabyrinthMatrix extends LabyrinthImpl {
-
-	private int matrix[][];
-	private int  freeValue, wallValue;
-
-	public LabyrinthMatrix(int rows, int columns) {
-		this.rows = rows;
-		this.columns = columns;
-
-		matrix = new int[rows][columns];
-	}
-
-	public LabyrinthMatrix() {
-	}
-
-	public void loadFromFile(String path) {
-		Scanner sc;
-		try {
-			sc = new Scanner(new File(path));
-			rows = sc.nextInt();
-			columns = sc.nextInt();
-			matrix = new int[rows][columns];
-			for (int i = 0; i < rows; i++)
-				for (int j = 0; j < columns; j++) {
-					matrix[i][j] = sc.nextInt();
-					if (matrix[i][j] == startValue)
-						start = new Position(i, j);
-					if (matrix[i][j] == finishValue)
-						finish = new Position(i, j);
-				}
-			sc.close();
```

Comment:
Hmm... I *think* that the Java way to do this would be to close the scanner in a "finally" block, so that if an exception is thrown *after* the scanner has been created, it would still be closed.